internal/vendors/equinix/equinix.go

```diff
@@ -5,159 +5,80 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"path"
 	"strings"
 	"time"
 
 	"github.com/google/go-github/v53/github"
-	"github.com/metal-toolbox/firmware-syncer/internal/config"
-	"github.com/metal-toolbox/firmware-syncer/internal/inventory"
 	"github.com/metal-toolbox/firmware-syncer/internal/vendors"
-	"golang.org/x/oauth2"
-
 	"github.com/pkg/errors"
-	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+
 	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
 )
 
 const GithubDownloadTimeout = 300
 
-// Equinix implements the Vendor interface methods to retrieve Equinix OpenBMC firmware files
-type Equinix struct {
```

Comment:
Now that the equinix vendor is gone, it'd make sense to change the package name and the filename to `github` and `github/github.go` respectively

Reply:
That makes sense. Thanks for the suggestion! Done.